graphicscontext.go

```diff
@@ -142,7 +142,6 @@ func (c *graphicsContext) Update(afterFrameUpdate func()) error {
 			if err != nil {
 				return err
 			}
-			defer f.Close()
```

Comment:
Why did you remove this?

Reply:
you're already closing it a few lines below

Reply:
`Close` is not called when `Encode` fails. This just ensures `Close` is called.

Reply:
ye, sorry didnt think about this